hpy/devel/src/runtime/ctx_type.c

```diff
@@ -35,7 +43,13 @@
         double _m_double;               \
         long double _m_longdouble;      \
         void *_m_pointer;               \
-    };
+    }
+
+typedef MAX_ALIGN_T _HPy_MaxAlign_t;
```

Comment:
nitpick: if this is a typedef maybe it should not be in uppercase?
Why do you even need it instead of just using `_HPy_MaxAlign_t`? (just curious)

Reply:
Right, doesn't make sense to have it at all. I think I wanted to have `#define SIZEOF_MAX_ALIGN_T`  but then realized that this would need to be generated in a separate step.